testdata/evaluations.yml

```diff
@@ -0,0 +1,59 @@
+- expression: 'not x'
```

Comment:
@aslakhellesoy I spent some time working on this branch. I'm running into the assumption in the Perl code that tags start with an '@' sign, but in these tests, they don't. Should I remove the check for the '@'-sign to distinguish junk from tags?

Reply:
I don’t think the parser should require operands to start with an @. The library is a general-purpose parser/evaluator for boolean infix expressions.

Reply:
Ok. I'll change the Perl version of the library.

One follow-up question: I noticed you don't have a test for the following expressions:

```
a and or
```
is the `or` on that line an operator or and operand?